integration-tests/src/test/java/oracle/kubernetes/operator/ItModelInImageConfigUpdate.java

```diff
@@ -425,6 +425,179 @@ public void testMiiConfigUpdateJdbcImage() {
     LoggerHelper.getLocal().log(Level.INFO, "SUCCESS - " + testMethodName);
   }
 
+  /**
+   * Create a domain without a JDBC DS using model in image and having configmap
```

Comment:
I'm not yet understanding the description of the initial state.  If the domain configuration does not have a JDBC DS, why is there a need for a config map?  What part of the model, if any, is in the config map at the beginning?

Reply:
The model file when creating the domain first looks like this:

domainInfo:
    AdminUserName: '@@SECRET:__weblogic-credentials__:username@@'
    AdminPassword: '@@SECRET:__weblogic-credentials__:password@@'
    ServerStartMode: 'prod'

topology:
    Name: domain1
    AdminServerName: "admin-server"
    Cluster:
        "cluster-1":
            DynamicServers:
                ServerTemplate:  "cluster-1-template"
                ServerNamePrefix: "managed-server"
                DynamicClusterSize: 5
                MaxDynamicClusterSize: 5
                CalculatedListenPorts: false
    Server:
        "admin-server":
            ListenPort: 7001
    ServerTemplate:
        "cluster-1-template":
            Cluster: "cluster-1"
            ListenPort : '@@PROP:MANSVR_PORT@@'
appDeployments:
    Application:
        myear:
            SourcePath: wlsdeploy/applications/sample_app.ear
            ModuleType: ear
            Target: 'cluster-1'

Reply:
very basic things to create a domain